mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/model/spi/PojoPropertyModel.java

```diff
@@ -23,6 +23,6 @@
 	 */
 	PojoGenericTypeModel<T> getTypeModel();
 
-	PropertyHandle getHandle();
+	PropertyHandle<? super T> getHandle();
```

Comment:
I think this should be `PropertyHandle<T>`.

I know that will cause problems in `GenericContextAwarePojoPropertyModel`; I'll add another comment there.